test/jdk/tools/jar/ValidatorTest.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright (c) 2025, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

One year is enough here, right?